official-build.props

```diff
@@ -1,2 +1,3 @@
 DISABLE_AUTOTAGGING=true
-USE_OPEN_SOURCE_SETTINGS=false
\ No newline at end of file
+USE_OPEN_SOURCE_SETTINGS=false
+SEPG_BUILD_ENV_IMAGE=cafapi/opensuse-jdk8-maven:1.0.0
```

Comment:
Not sure if we want to set SEPG_BUILD_ENV_IMAGE? We didn't for the job-service as you though there may have been other changes required for the job-service to build on linux:

https://portal.digitalsafe.net/browse/SCMOD-11252?focusedCommentId=3025235&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3025235

Reply:
If it works then that's great.